rest/service/dashboardTemplate.go

```diff
@@ -84,6 +84,14 @@ var (
 			Module:   "./OpenShiftWidget",
 			Defaults: models.BaseWidgetDimensions.InitDimensions(models.BaseWidgetDimensions{}, 2, 2, 4, 1),
 		},
+		models.RecentlyVisited: models.ModuleFederationMetadata{
```

Comment:
@aferd I've also been trying to keep our readme up to date with the various widgets. Do you mind adding `RecentlyVisited` here (to the `AvailableWidgets` example? https://github.com/RedHatInsights/chrome-service-backend/blob/main/docs/dashboard-layouts.md#widgets-mapping